vtools/cli.py

```diff
@@ -31,11 +31,26 @@
               help="Sample(s) in positive-vcf to consider. "
                    "May be called multiple times",
               required=True)
-def evaluate_cli(call_vcf, positive_vcf, call_samples, positive_samples):
+@click.option("-s", "--stats", type=click.Path(writable=True),
+              help="Path to output stats json file", default='-')
```

Comment:
I am glad '-' works. using `sys.stdout` is maybe more specific, but this maybe some click related stuff.